static/app/components/onboarding/useMutateOnboardingTasks.spec.tsx

```diff
@@ -32,14 +29,8 @@ describe('useMutateOnboardingTasks', () => {
       body: testTask,
     });
 
-    const {result} = renderHook(() => useMutateOnboardingTasks(), {
-      wrapper: ({children}) => (
-        <OrganizationContext value={organization}>
-          <QueryClientProvider client={makeTestQueryClient()}>
-            {children}
-          </QueryClientProvider>
-        </OrganizationContext>
-      ),
+    const {result} = renderHookWithProviders(() => useMutateOnboardingTasks(), {
```

Comment:
Should we do this kind of think now, or in a followup?

```suggestion
    const {result} = renderHookWithProviders(useMutateOnboardingTasks, {
```